create-snowpack-app/app-template-blank-typescript/src/index.ts

```diff
@@ -1,11 +1,9 @@
-/**
- * This file is just a silly example to show everything working in the browser.
- * When you're ready to start on your site, clear the file. Happy hacking!
- **/
+const counter = document.querySelector("#counter") as HTMLSpanElement;
+let seconds = 0;
 
-import confetti from 'canvas-confetti';
+setInterval(() => {
+  seconds += 1;
+  counter.textContent = seconds.toString();
+}, 1000);
 
-confetti.create(document.getElementById('canvas') as HTMLCanvasElement, {
-  resize: true,
-  useWorker: true,
-})({ particleCount: 200, spread: 200 });
+export {};
```

Comment:
I don’t think we need this

Reply:
I added it after TS complained that `'index.ts' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module.`

The comment above the `isolatedModules` option in `tsconfig.json` says "more strict checking for errors that per-file transpilers like `esbuild` would crash", so I'm assuming the option is there for a reason.

Reply:
Ah that’s…worth looking into. OK I’m fine with leaving and I can investigate more later. Thanks!